test/test_filesystem.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import os.path
+import picard.formats
+import unittest
+import shutil
+from contextlib import suppress
+
+from PyQt5 import QtCore
+from picard import config
+from tempfile import mkdtemp
+from test.test_formats import FakeTagger
+
+
+settings = {
+    'enabled_plugins': '',
+    'move_files': True,
+    'move_additional_files': True,
+    'move_additional_files_pattern': 'cover.jpg',
+}
+
+
+class TestFileSystem(unittest.TestCase):
+
+    def setUp(self):
+        self.src_directory = mkdtemp()
+        self.tgt_directory = mkdtemp()
+        QtCore.QObject.tagger = FakeTagger()
+        config.setting = settings.copy()
+
+    def tearDown(self):
+        shutil.rmtree(self.src_directory)
+        shutil.rmtree(self.tgt_directory)
+        pass
```

Comment:
Leftovers?

Reply:
This is to remove the temporary files from the tests